components/Moola.vue

```diff
@@ -43,11 +43,6 @@
             value: {
                 type: [ Number, String ],
                 default: 0
-            },
-
-            delimiter: {
```

Comment:
Why are you removing support for custom delimiters?

Reply:
Copy and paste error i believe